.editorconfig

```diff
@@ -0,0 +1,25 @@
+[*]
+charset = utf-8
+end_of_line = lf
+ij_continuation_indent_size = 8
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 100
+
+[*.xml]
+indent_size = 2
+
+[pom.xml]
+max_line_length = 140
```

Comment:
not sure why 140, we have lines loner 140 in pom.xml.
please share details.

Reply:
any violation should be found by the plugin https://github.com/checkstyle/checkstyle/pull/16598

Reply:
then we should remove it. thanks